docs/getting-started/Installation.stories.mdx

```diff
@@ -43,6 +43,29 @@ To import Orbit experimental components styles, also add the following import de
 @import "@orbit-ui/experimental/index.css";
 ```
 
+## Set up your tooling environment
+
+Orbit offer an ESLint plugin to see in-context help in your IDE. This includes accessibility pointers, deprecation notices, and other helpful tips.
+
+First, add the ESlint plugin to your dependencies:
+
+<PackageInstallationSnippet workspaceFolder="tooling" packageName="eslint-plugin" />
+
+Second, add the plugin to your ESlint config. For example, your .eslintrc.json file may look like this:
```

Comment:
.eslintrc.json -> `.eslintrc.json`

Reply:
Not a big deal but since I wrote a majority of the documentation and prefered "Then" to "Second", to keep the same tone accross our documentation site, it might be a good idea to prefer "Then".

Therefore, "Second" -> "Then"